tests/drivers/uart/uart_async_api/src/test_uart.h

```diff
@@ -57,6 +57,7 @@
 #define UART_DEVICE_NAME DT_LABEL(DT_NODELABEL(lpuart2))
 #elif defined(CONFIG_BOARD_MIMXRT1050_EVK) || \
 	defined(CONFIG_BOARD_MIMXRT1060_EVK) || \
+	defined(CONFIG_BOARD_MIMXRT1060_EVKB) || \
```

Comment:
Non blocking suggestion- I believe you need to add a board specific overlay for `mimxrt1060_evkb` if you want to enable LPUART3 for this test, and relocate sram into DTCM to avoid caching issues.

Reply:
Added an overlay for this